theme/snippets/news-feed-item.liquid

```diff
@@ -1,9 +1,9 @@
-<div class="NewsFeedItem overflow-hidden overflow-x-scroll flex items-center border-thick-bottom-black py1_25 md:py_75 xs:pr0">
-  <div class="flex col-1 md:mx_75">
+<a href="{{ link }}" target="_blank" rel="noreferrer noopener" class="NewsFeedItem flex items-center py1_25 md:py_75 xs:pr0 border-thick-bottom-black">
+  <div class="flex col-2 lg:col-1 md:mx_75">
     <img class="NewsFeedItem__image fit-contain w100 h100" src="{{ image_src }}" alt="{{ image_alt }}"/>
   </div>
-  <div class="flex items-center justify-between col-11">
-    <span class="NewsFeedItem__body serif px1">{{ body.html }}</span>
-    <span class="NewsFeedItem__third-column sans-xs sans-serif md:pl2">{{ third_column.html }}</span>
+  <div class="flex items-center justify-between col-10 lg:col-11">
+    <span class="NewsFeedItem__body serif px1 col-9 md:col-11">{{ body }}</span>
+    <span class="NewsFeedItem__cta-label sans-xs sans-serif md:pl2 col-3 md:col-1 flex flex-col items-end">{{ cta_label }}</span>
   </div>
-</div>
+</a>
```

Comment:
Changed this from markdown to text and called it `cta_label` like Jake mentioned. 
For accessibility reasons, I made `cta_label` text instead of a link because then there would be the row (a link) and the cta link inside of a link.